### ISSUE TITLE
Feature Annotations Below Track

### ISSUE THREAD

**Issue by smk508** [Edinburgh-Genome-Foundry/DnaFeaturesViewer#47]
Hi, is it possible to have feature annotations appear below a plot? I have a figure like this, with a genome track underneath another plot for visualization, and I would like the text boxes to go below the track if possible:

![image](https://user-images.githubusercontent.com/13842719/96042011-bb685100-0e3a-11eb-95eb-2aab51ea9011.png)

**Comment by veghp** [comment]
@smk508  would this example be a solution for your question? https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer/pull/95#issuecomment-3019494661

**Comment by smk508** [comment]
Yes, that's exactly it! I'm sure people will find it useful. I ended up making the figure manually by splicing different figures together in Adobe and this is how it turned out (fig 3) https://pmc.ncbi.nlm.nih.gov/articles/PMC9528964/

**Comment by veghp** [comment]
Thanks, I'll add that code to the examples then.
